### PR TITLE
Fix linting issues

### PR DESCRIPTION
**Pull request by enolfc**
# Summary

Keep fixing format of files

### CODE REVIEW

**Comment by gwarf** [comment]
Cool, so now https://github.com/gaurav-nelson/github-action-markdown-link-check and https://github.com/tcort/markdown-link-check will become your firends :)

**Comment by gwarf** [comment]
For https://github.com/EGI-Foundation/documentation/pull/145/checks?check_run_id=1424507155#step:4:64 we probably will have to add it to the list of excluded stuff.

**Comment by enolfc** [comment]
> https://github.com/EGI-Foundation/documentation/pull/145/checks?check_run_id=1424507155#step:4:64

in fact I tried to add this as an exception but apparently did it the wrong way, retrying...

**Comment by enolfc** [comment]
Now things look good, @gwarf can you check if things are ok?

File: content/en/providers/cloud-compute/_index.md [hunk 1]
```diff
@@ -5,44 +5,42 @@ weight: 30
 type: "docs"
 ---
 
-This documentation covers how to join the EGI Cloud federation as a provider.
-If you are interested in joining please first contact EGI operations team
-(``operations _at_ egi.eu``), expressing interest and providing few details about:
+This documentation covers how to join the EGI Cloud federation as a provider. If
+you are interested in joining please first contact EGI operations team
+(`operations _at_ egi.eu`), expressing interest and providing few details about:
```
Comment (gwarf) [inline content/en/providers/cloud-compute/_index.md:10]: So at various places we have real email, like for the support page where we have:
```markdown
[support@egi.eu](mailto:support@egi.eu)
```
We probably want to have a coherent way of dealing with this.
I'm quite fine with having the real emails, they are probably already here and there on the net.

File: content/en/providers/cloud-compute/_index.md [hunk 2]
```diff
@@ -5,44 +5,42 @@ weight: 30
...
 
-* the projects you may be involved in as cloud provider
+- the projects you may be involved in as cloud provider
 
-* the user communities you want to support (a.k.a. Virtual Organisations, VO).
+- the user communities you want to support (a.k.a. Virtual Organisations, VO).
```
Comment (gwarf) [inline content/en/providers/cloud-compute/_index.md:14]:
```suggestion
- the user communities you want to support (AKA Virtual Organisations, VO).
```

AKA seems to be more frequent: https://en.wiktionary.org/wiki/AKA#English

**Review by gwarf — APPROVED** [review]
LGTM!
(still added some small comments)

File: content/en/providers/cloud-compute/_index.md (hunk 1 above)
Comment (enolfc) [reply content/en/providers/cloud-compute/_index.md]: Fixed

**Comment by enolfc** [comment]
Merging considering the latest changes were the small suggestions  above

**Pull request merged**